catalog-entities/marketplace/packages/backstage-plugin-techdocs-module-addons-contrib.yaml

```diff
@@ -35,3 +33,33 @@ spec:
             backstage.plugin-techdocs-module-addons-contrib:
               techdocsAddons:
                 - importName: ReportIssue
+    - title: ReportIssue configuration
+      content:
+        dynamicPlugins:
+          frontend:
+            backstage.plugin-techdocs-module-addons-contrib:
+              techdocsAddons:
+                - importName: ReportIssue
```

Comment:
Looks like you may have missed the expandable navigation addon?

https://backstage.io/docs/reference/plugin-techdocs-module-addons-contrib.expandablenavigation/